### PR TITLE
[CLOUDTRUST-2296] Add a method on account API to send execute actions email

### PR DESCRIPTION
**Pull request by fperot74**

### CODE REVIEW

File: keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java [hunk 1]
```diff
@@ -86,4 +100,53 @@ public Response deleteAccount() {
...
+            throw new WebApplicationException(
+                ErrorResponse.error("User is disabled", Status.BAD_REQUEST));
+        }
+
+        if (lifespan == null) {
+            lifespan = realm.getActionTokenGeneratedByAdminLifespan();
+        }
+        int expiration = Time.currentTime() + lifespan;
+        String redirectUri = null;
+        String clientId = Constants.ACCOUNT_MANAGEMENT_CLIENT_ID;
```
Comment (harture) [inline keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java]: I don't understand where does this Constants come from ... I don't see it in imports neither in package. What am I missing ?

File: keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java [hunk 2]
```diff
@@ -86,4 +100,53 @@ public Response deleteAccount() {
...
+
+            String link = builder.build(realm.getName()).toString();
+
+            this.session.getProvider(EmailTemplateProvider.class)
+              .setAttribute(Constants.TEMPLATE_ATTR_REQUIRED_ACTIONS, token.getRequiredActions())
+              .setRealm(realm)
+              .setUser(user)
+              .sendExecuteActions(link, TimeUnit.SECONDS.toMinutes(lifespan));
+
+            //audit.user(user).detail(Details.EMAIL, user.getEmail()).detail(Details.CODE_ID, accessCode.getCodeId()).success();
```
Comment (harture) [inline keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java]: To remove  and/or convert into standard event

File: keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java [hunk 3]
```diff
@@ -86,4 +100,53 @@ public Response deleteAccount() {
...
+
+            //adminEvent.operation(OperationType.ACTION).resourcePath(session.getContext().getUri()).success();
```
Comment (harture) [inline keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java]: To remove ?

File: keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java (hunk 1 above)
Comment (fperot74) [reply keycloak-rest-api-extensions/src/main/java/io/cloudtrust/keycloak/services/resource/api/account/FixedAccountRestService.java]: It comes from org.keycloak.models
(the code to send the mail is mainly copied from Keycloak)

**Pull request merged**